website/server/controllers/api-v3/user.js

```diff
@@ -1045,17 +1047,17 @@ api.userSell = {
 };
 
 /*
-* @api {post} /api/v3/user/unlock Unlocks items by purchase.
+* @api {post} /api/v3/user/unlock Unlock item or set of items by purchase
 * @apiVersion 3.0.0
 * @apiName UserUnlock
 * @apiGroup User
 *
 * @apiParam {string} path Query parameter. The path to unlock
 *
 * @apiSuccess {Object} data.purchased
-* @apiSuccess {Object} data.items`
-* @apiSuccess {Object} data.preferences`
-* @apiSuccess {string} message`
+* @apiSuccess {Object} data.items
+* @apiSuccess {Object} data.preferences
+* @apiSuccess {string} message
```

Comment:
I'm not sure what those backticks were for. You might want me to revert that change or modify it in some other way.